spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/Evaluator.scala

```diff
@@ -88,7 +89,12 @@ class Evaluator[T: ClassTag] private[optim](model: Module[T])(implicit ev: Tenso
            vMethods: Array[ValidationMethod[T]]
           ): Array[(ValidationResult, ValidationMethod[T])] = {
 
-    val dummyInput = dataset.takeSample(withReplacement = false, num = 1).head.getInput()
+    // TODO this should be fixed next release because of some objects can't be serialized
```

Comment:
no need. We are woking on fixing it. Just remind minibatch should be serializable.